packages/sdk-py/src/agent_relay/client.py

```
@@ -348,6 +349,7 @@ async def spawn_pty(
         }
         if task is not None: payload["task"] = task
         if model is not None: payload["model"] = model
+        if harness is not None: payload["harness"] = harness
```

Ruff (0.15.13)

[error] Multiple statements on one line (colon)

(E701)